packages/shared/src/uops/protocols/eip-712/const.ts

```diff
@@ -0,0 +1 @@
+export const EIP712_TX_TYPE = 113
```

Comment:
is there a need for seperate file?

Reply:
wanted to preserve the same structure as for all other protocols